site/content/en/docs/concepts/multikueue.md

```diff
@@ -60,7 +60,7 @@ Known Limitations:
 
 #### MultiKueueBatchJobWithManagedBy enabled
 
-When you want to submit Job to a ClusterQueue with a MultiKueue admission check, you should set the `spec.managedBy` field to `kueue.x-k8s.io/multikueue`, otherwise the admission check controller will `Reject` the workload causing it to be marked as `Finished` with an error indicating the cause.
+When you want to submit Job to a ClusterQueue with a MultiKueue admission check, you should set the `spec.managedBy` field to `kueue.x-k8s.io/multikueue`, otherwise the admission check controller will `Reject` the workload.
```

Comment:
this was also no longer the truth, I allowed myself to change it in this PR as we basically updates the same topic

Reply:
sgtm